build/registry/check.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright (C) 2022 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+)
+
+var (
+	forbiddenSourceIDs   = []uint{0}
+	forbiddenSourceNames = []string{"syscall", "internal", "plugins"}
```

Comment:
It would be better to fetch that list from the `registry.yaml` file. 
Anyway, not a blocker for merging this PR.